python/ooi_data_explorations/profilers.py

```diff
@@ -221,3 +221,75 @@ def updown(db, db_level):
         dzdt[pks[i]:pks[i + 1]+1] = dz[i]
 
     return pks, dzdt
+
+
+def pair_profiles(ds, break_t = 60):
+    """
+    Function to pair profiles with their partner
+    profile.
+    
+    :param ds: the profile dataset with the profile variable added
+    :param break_t: the time (min) separation below which a profile is paired
+    :return ds: the profile dataset with the paired profile variable added
+    """
+
+    parameters = [p for p in ds.variables]
+    if 'profile' not in parameters:
```

Comment:
Instead of raising an error, why not add code that would add the profile parameter?